CONTRIBUTING.POSTGRESQL.CHEATSHEET.md

```diff
@@ -0,0 +1,176 @@
+`yarn test-integration` | **Type**: Partial test<br>**Requires**: running db
+`yarn test-unit` | **Type**: Partial test<br>**Requires**: no requirement can be run as-is
+`DIALECT=postgres npx mocha test/**/query-generator.test.js` | **Type**: Specific test<br>**Notice**: it calls `mocha` directly <br>**Requires**: varies based on test
+
+---
+
+## Debugging / Cheatsheet
+
+Command | Description
+--- | ---
+`console.log` | `node --inspect` is not currently utilized and would require modifications into the tests. In the meantime, use `console.log` for debugging
+`docker ps -a` | Display running containers
+`docker exec -it <container name> psql -d sequelize_test -U sequelize_test` | Connect to a running database inside the container and run psql (psql is PostgreSQL-only)
+
+### PgAdmin4
+1. Running pgadmin4 as a container (pgadmin is PostgreSQL-only)
+
+   Create the container:
+   ```bash
+   docker run -d --name pgadmin4 -p 8888:80 -e 'PGADMIN_DEFAULT_EMAIL=test@example.com' -e 'PGADMIN_DEFAULT_PASSWORD=sequelize_test' dpage/pgadmin4
+   ```
+
+   Once the postgres container and the pgadmin container are both running, it's only a matter of ensuring they're on the same network.  
+   
+   Existing networks may be found with `docker network ls`:
+   ```bash
+   NETWORK ID     NAME                            DRIVER    SCOPE
+   df4705da72d2   bridge                          bridge    local
+   081ed5b71d99   host                            host      local
+   c22cad586f28   none                            null      local
+   092ed56f9f1b   sequelize-postgres-10-network   bridge    local
+   ```
+   
+   The drivers with _bridge_ are of particular interest. More details can be found using `docker inspect`; for instance, `docker inspect sequelize-postgres-10-network`, reveals the containers bound to it:
+
+   ```json
+      ...
+      "Containers": {
+            "8544cdd984ebca72b8c819ca4195f6d130807fe7a9c2a49cb389926f2c2799d8": {
+                "Name": "sequelize-postgres-10",
+                "EndpointID": "ae0f4df1a94b16b7316d27b086f6e3506a06d692282272e5536762e1053a78f0",
+                "MacAddress": "02:42:ac:16:00:02",
+                "IPv4Address": "172.22.0.2/16",
+                "IPv6Address": ""
+            }
+        },
+      ...
+   ```
+
+   The idea is to have a network bridge that includes both containers. There are two ways to accomplish this:
+
+   1. Add one of the containers to their existing bridges. Using the network we discovered above we can run:
+      ```bash
+      docker network connect sequelize-postgres-10-network pgadmin4
+      ```
+      This will add pgadmin's container (named _pgadmin4_) to the postgres container's network.
+
+   1. Create a new network and add containers. If your preference is to leave the containers in tact, you may want to create a new bridge network and then connect the containers to it. Below we call this network `pg-net`, but it could be named whichever:
```

Comment:
```suggestion
   1. Create a new network and add containers. If your preference is to leave the containers intact, you may want to create a new bridge network and then connect the containers to it. Below we call this network `pg-net`, but it could be named whichever:
```